book/src/proposals/tower-lockout-slashing.md

```diff
@@ -26,6 +27,11 @@ prover needs to submit two votes.
 * vote 3: 0, 1, 2, 3
 * vote 4: 0, 1, 2, 5
 
+The second vote skips 3, and therefore allows the vote to be accepted
```

Comment:
If I see two votes:

vote 1: 0, 1, 2, 3
vote 2: 0, 1, 2, 6

This is not a slashing condition, even if the the validator submitted vote 2 first, then submitted vote 1. Technically they are voting on two different forks, but they can't build a very deep tower on fork 1 without getting slashed. Hopefully there's no correctness problem/attack vector we're missing here

Reply:
@carllin yea, that's correct.  Because there is no way to know which one came first.  3 or 6.